cmd/templatizer/app/flags.go

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) 2020 The Jaeger Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package app
+
+import (
+	"github.com/spf13/cobra"
+)
+
+// Options represent configurable parameters for jaeger-templatizer
+type Options struct {
+	Mapping   string
+	EsVersion int64
+	Shards    int64
+	Replicas  int64
+	EsPrefix  string
+	UseILM    string // using string as util is being used in python and using bool leads to type issues.
+}
+
+const (
+	mappingFlag   = "mapping"
+	esVersionFlag = "esVersion"
+	shardsFlag    = "shards"
+	replicasFlag  = "replicas"
+	esPrefixFlag  = "esPrefix"
+	useILMFlag    = "useILM"
+)
+
+// AddFlags adds flags for templatizer main program
+func (o *Options) AddFlags(command *cobra.Command) {
+	command.Flags().StringVarP(
+		&o.Mapping,
+		mappingFlag,
+		"m",
```

Comment:
nit: we tend to avoid using cobra's flags, and instead use stdlib flags (which does not have single-letter variants, which are just maintenance burden)

Reply:
I have followed anonymizer to build this cli - as suggested by @albertteoh, where we do use cobra. I also felt cobra way was more cleaner.

We can remove single letter short-hands (they are optional), we are not using them anyways. Thoughts?

Reply:
ok. If we can remove one-letter ones that it's ok to keep using cobra.